tests/acceptance/01_vars/02_functions/classfilterdata_array_of_objects_foo.cf

```diff
@@ -0,0 +1,78 @@
+body common control
+{
+  bundlesequence => { "check" };
+}
+
+bundle agent test(context, index)
+{
+  meta:
+    "description" -> { "ENT-6193", "CFE-3421" }
+      string => "Test for expected results from policy function classfilterdata() with array of objects using context '$(with)' and key $(key)",
```

Comment:
here do you need to write index instead of key? I don't see `$key` defined anywhere.